jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/skaffold/SkaffoldInitMojo.java

```diff
@@ -37,12 +38,17 @@
   @Override
   public void execute() throws MojoExecutionException {
     checkJibVersion();
+    MavenProject project = getProject();
+    // Ignore parent projects
+    if (project.getModules().size() > 0) {
```

Comment:
So this was another bug, right?

Reply:
I guess you could call it that. It was something I was aware of but didn't think was necessary to change until now.